Assets/MRTK/StandardAssets/EditorUtilities/OnLoadUtilities.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    [InitializeOnLoad]
+    static class OnLoadUtilities
+    {
+        private const string SessionStateKey = "StandardAssetsOnLoadUtilitiesSessionStateKey";
+
+        private const string ShaderSentinelGuid = "05852dd420bb9ec4cb7318bfa529d37c";
+        private const string ShaderSentinelFile = "MRTK.Shaders.Sentinel";
+
+        private const string ShaderImportDestination = "MRTK/Shaders";
```

Comment:
Do we want this to be MixedRealityToolkit.Shaders for symmetry with MixedRealityToolkit.Generated?

Reply:
We could. I picked a MRTK/Shaders to avoid long path issues. The generated folder was left as-is in 2,4 to avoid possible compat and source control issues.